<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1140</width>
    <height>840</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1140</width>
    <height>840</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1920</width>
    <height>1080</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#Form{
background-image: url(C:/Users/deadb/Documents/GitHub/AvailData-Public/Client/login_photos/temp_main_background.jpg);
background-position: center;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QFrame" name="frame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>943</width>
       <height>616</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>1100</width>
       <height>800</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">*{
background:rgba(0,0,0,0.8);
background-color: rgb(49, 51, 56);
font-family: Trebuchet MS;
color: white;
}

QFrame
{
background: transparent;
}

QLabel
{
background-color:rgba(0,0,0,0.5);
border-radius: 15;
color: white;
}

QComboBox{
border-radius: 10px;
background-color: rgba(49, 51, 56,0.8);
font-size:19px;
text-align: center;
}
</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="selected_table_label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>381</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Trebuchet MS</family>
        <pointsize>-1</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(0,0,0,0.8);
font-size:19px;</string>
      </property>
      <property name="text">
       <string>  Selected Table</string>
      </property>
     </widget>
     <widget class="QComboBox" name="table_select_combobox">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>30</y>
        <width>221</width>
        <height>41</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 10px;
background-color: rgba(49, 51, 56,0.8);
font-size:19px;</string>
      </property>
      <property name="editable">
       <bool>false</bool>
      </property>
      <property name="currentText">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="result_select_label">
      <property name="geometry">
       <rect>
        <x>860</x>
        <y>20</y>
        <width>221</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Trebuchet MS</family>
        <pointsize>-1</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(0,0,0,0.8);
font-size:19px;</string>
      </property>
      <property name="text">
       <string> Results  </string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QComboBox" name="result_select_combobox">
      <property name="geometry">
       <rect>
        <x>870</x>
        <y>30</y>
        <width>81</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Trebuchet MS</family>
        <pointsize>-1</pointsize>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="editable">
       <bool>false</bool>
      </property>
      <property name="insertPolicy">
       <enum>QComboBox::NoInsert</enum>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QComboBox::AdjustToContentsOnFirstShow</enum>
      </property>
      <item>
       <property name="text">
        <string>25</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>50</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>100</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>250</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>500</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>1000</string>
       </property>
      </item>
     </widget>
     <widget class="QFrame" name="frame_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>100</y>
        <width>1061</width>
        <height>611</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame
{
border-radius: 20;
background-color: rgba(0,0,0,0.8);
font-size: 15px;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QTableWidget" name="loaded_table_edit">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>1021</width>
         <height>551</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">
QTableView::item:selected {
    color: white;
    background-color: black;
	font-size: 15px;
}

QTableWidget {
	font-size: 15px;
    background-color: rgba(49, 51, 56, 0.7);
    alternate-background-color: rgba(25, 28, 29, 0.8);
    border-radius: 0px;
}

QHeaderView::section {
    font-size: 18px;
    background-color: black;
    border-radius: 0px;
}

QTableCornerButton::section {
    background-color: black;
}

QTableCornerButton::section:selected {
    color: white;
    background-color: black;
}

QTableView::item {
    background-color: rgba(60, 63, 66, 0.5);
}

QScrollBar{
	color: white;
    background-color: black;
}

</string>
       </property>
       <property name="sizeAdjustPolicy">
        <enum>QAbstractScrollArea::AdjustToContents</enum>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="gridStyle">
        <enum>Qt::SolidLine</enum>
       </property>
       <property name="sortingEnabled">
        <bool>false</bool>
       </property>
       <attribute name="horizontalHeaderVisible">
        <bool>true</bool>
       </attribute>
       <attribute name="horizontalHeaderCascadingSectionResizes">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderCascadingSectionResizes">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderHighlightSections">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderStretchLastSection">
        <bool>false</bool>
       </attribute>
      </widget>
      <widget class="QRadioButton" name="readwrite_radioButton">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>930</x>
         <y>10</y>
         <width>111</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QRadioButton:enabled{
font-size: 16px;
border-radius: 10px;
padding-left: 10px;
background-color: black;
color: white;
}

QRadioButton::indicator {
width: 11px;
height: 11px;
border-radius: 5px;
}

QRadioButton::indicator::unchecked{ 
border: 1px solid; 
border-color: rgb(132,132,132);
border-radius: 6px;
background-color: white; 
width: 11px; 
height: 11px; 
}

QRadioButton::indicator::checked{ 
border: 3px solid; 
border-color: white;
border-radius: 6px;
background-color: red; 
width: 7px; 
height: 7px; 
}

QRadioButton:disabled{
font-size: 16px;
border-radius: 10px;
padding-left: 4px;
background-color: black;
color: white;
}

QRadioButton::indicator:disabled {
width: 0px;
height: 0px;
border-radius: 0px;
border: 0px;
}</string>
       </property>
       <property name="text">
        <string>Edit Mode</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="commit_pushButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>810</x>
         <y>10</y>
         <width>111</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton:enabled{
font-size: 16px;
border-radius: 10px;
background-color: black;
color: white;
}


QPushButton:disabled{
font-size: 16px;
border-radius: 10px;
background-color: transparent;
color: transparent;
}</string>
       </property>
       <property name="text">
        <string>Save Changes</string>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="selected_table_label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>730</y>
        <width>1061</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Trebuchet MS</family>
        <pointsize>-1</pointsize>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(0,0,0,0.7);
font-size:19px;</string>
      </property>
      <property name="text">
       <string>Functionality Space</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="logout_button">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>740</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
border-radius: 10;
color: red;
font-size:19px;
}

QPushButton:hover
{
color: black;
border-radius:13px;
background: rgb(220, 50, 50);
font-size:20px;
}</string>
      </property>
      <property name="text">
       <string>Log Out</string>
      </property>
     </widget>
     <widget class="QLabel" name="title_label">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>20</y>
        <width>431</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Trebuchet MS</family>
        <pointsize>-1</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 38px;
color: red;
background-color: rgba(0,0,0,0.8);</string>
      </property>
      <property name="text">
       <string>Temporary Main Page</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="lastfirst_pushButton">
      <property name="geometry">
       <rect>
        <x>950</x>
        <y>30</y>
        <width>51</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Trebuchet MS</family>
        <pointsize>-1</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
font-size: 19px;
}</string>
      </property>
      <property name="text">
       <string>Last</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
